script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_4_1A.txt

```diff
@@ -77,51 +77,51 @@ I'm sorry. As a human being, this is beyond anything I can imagine.
 <Page23>
 "It was only a couple of hours ago that I found out from the municipal records that your clothes belonged to that school.
-I was going to wait until nightfall to catch you, but I didn't want to wait another minute. So in the end, I decided to wait over there."
+I was going to wait until nightfall to catch you, but I was too impatient. In the end, I decided to wait over there."
 <Page24>
 "........"
 I see... so that's why she was waiting for me in front of the school.
 I'm sure that after experiencing that much pain, she must have been dying to kill me.
 But...
 <Page25>
-"...I don't understand. If you hated me so much, why did you say that you will forgive me?"
+"...I don't understand. If you hated me so much, why did you say that you'd forgive me?"
 <Page26>
 "―――Good point.
-To put it simply, I guess I cooled off over time...or something.
+To put it simply, I guess I cooled off over time... or something.
 I've already used up most of my power, and instead of just mindlessly killing you, I figured it would be more efficient to let you become my meat shield."
 <Page27>
-"...Hey. Didn't you just say something awful just now?"
+"...Hey. Didn't you say something awful just now?"
 "Eh? Did I really say something like that?"
 "You just said something about using people as shields."
 Not to mention that scary image.
 <Page28>
-"What? Isn't that obvious.
+"What, isn't that obvious?
 Sure, you could say that I've forgiven you, but that was only because I had to deal with my own personal feelings.
 The act of murder that you committed must be atoned for by your actions, not by your feelings, right?"
 "I mean, that's true, but..."
 <Page29>
-I knew I was going to be killed for revenge, but becoming someone else's shield is a completely different issue.
+I was expecting to be killed in revenge, but becoming someone's shield is something else entirely. 
 ...Well, I can't really put into words why they're different, but I still have a bad feeling about it either way.
 <Page30>
 "I don't know if you're being honest or just being mean.
 In case you forgot, you were the one who killed me, remember?
-I know it may be hard to imagine, but it takes a lot of energy to come back to life after being dead."
+I know it may be hard to imagine, but it takes a lot of energy to come back to life after dying."
 <Page31>
 "It wouldn't have mattered if you'd merely killed me.
 But the way you cut me was nothing like I've ever seen before, and because I couldn't reconnect my body, I had no choice but to rebuild from scratch.
 As a result, I had to use practically all of my strength just to come back to life!"
 <Page32>
-The girl is so angry that she is positively fuming.
-Her anger from last night, forgotten until now, seems to have come back to mind during her ramble.
+The woman is so angry that she is positively fuming.
+Her anger from last night, forgotten until now, seems to have returned to mind during her ramble.
```

Comment:
I would drop `to mind` if we use `returned` here, since `come to mind` is more of a set phrase and `return to mind` isn't.